### DISCUSSION TITLE
How to set target currency for currency conversion in expression?

### DISCUSSION THREAD

**Discussion by MurzNN** [Qalculate/qalculate-gtk#397] [Q&A] [answered]
I need to set custom target currency for conversion directly in expression, but can't find in documentation how to do it, can anybody please explain? If it is not possible, is feature request about this exists already?

Example of possible expression:
```
100 USD as EUR
```

**Comment by hanna-kn** [comment, marked as answer]
`100 USD to EUR` (there may also be a translated variant), `100 USD->EUR`, or `100 USD➞EUR` (the symbol used when pressing the corresponding keypad button). "to" or arrow are used for any type of conversion (currency, unit, number base, time zone, etc.).
